packages/components/dashboard/details-card/details-card.scss

```diff
@@ -0,0 +1,28 @@
+.details-card-title {
+  font-weight: var(--pf-global--FontWeight--normal);
+  font-size: var(--pf-global--FontSize--lg);
+}
+
+.pf-c-card__header .pf-c-card__title {
+  padding-left: var(--pf-c-card--child--PaddingLeft);
+}
```

Comment:
no need to define same classes at two places (same are defined in status card as well)...
remove from both places and add common scss to dashboard.scss file...